<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-G2TLUC4" ignored="0" name="interface" tests="1" failures="1" timestamp="09 六月 2017 08:05:12 GMT" time="0.506" errors="0">
  <testcase name="test_method" time="0.491" classname="com.zzy.Interface.Test_Interface">
    <failure type="org.testng.internal.reflect.MethodMatcherException" message="
Data provider mismatch
Method: test_method([Parameter{index=0, type=[[Ljava.lang.Object;, declaredAnnotations=[]}])
Arguments: [(java.lang.String)Enable,(java.lang.String)NO,(java.lang.String)Name,(java.lang.String)Http,(java.lang.String)Path,(java.lang.String)Tpye,(java.lang.String)Input,(java.lang.String)Expect Output,(java.lang.String)Autal Output,(java.lang.String)error_code,(java.lang.String)error_message,(java.lang.String)Result]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
Data provider mismatch
Method: test_method([Parameter{index=0, type=[[Ljava.lang.Object;, declaredAnnotations=[]}])
Arguments: [(java.lang.String)Enable,(java.lang.String)NO,(java.lang.String)Name,(java.lang.String)Http,(java.lang.String)Path,(java.lang.String)Tpye,(java.lang.String)Input,(java.lang.String)Expect Output,(java.lang.String)Autal Output,(java.lang.String)error_code,(java.lang.String)error_message,(java.lang.String)Result]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
... Removed 5 stack frames]]>
    </failure>
  </testcase> <!-- test_method -->
</testsuite> <!-- interface -->
